scripts/core/EXT.rst

```diff
@@ -42,9 +64,9 @@ The following naming conventions must be followed for **standard** extensions:
 
 ## --validate=off
   - All extension functions must be postfixed with `Ext`
-  - All macros must use all caps `${X}_NAME_EXT` convention
-  - All structures, enumerations and other types must follow `${x}_name_ext_t` snake case convention
-  - All enumerator values must use all caps `${X}_ENUM_EXT_ETOR_NAME` convention
+  - All macros must use all caps <PREFIX>_NAME_EXT convention where PREFIX is `${X}_NAME_EXT` or `${S}_NAME_EXT` accordingly
```

Comment:
PREFIX is one of `${X}` or `${S}`, yielding `${X}_NAME_EXT` for a core macro and `${S}_NAME_EXT` for a sysman macro.

Reply:
@wdamon-intel not sure I follow, are you suggesting to reword the statement as you have it above?